AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -1981,6 +2006,21 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		057DE97CCEF5219E324C2A17 /* 📦 Copy Pods Resources */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "📦 Copy Pods Resources";
```

Comment:
I already saw that :( I'm on 1.0. I'm currently in the process to revert the project changes that has nothing to do with the change.